src/control/lib/control/pool.go

```diff
@@ -378,6 +378,8 @@ type (
 		Leader          uint32               `json:"leader"`
 		Rebuild         *PoolRebuildStatus   `json:"rebuild"`
 		TierStats       []*StorageUsageStats `json:"tier_stats"`
+		InfoBit         int32                `json:"info_bit"`
```

Comment:
Trying to understand the motivation for including this.

Reply:
The idea was to stick to the C code to easily be able to manage the evolution of this bitfield and to not add to much extra field to the RPC message.